brewtils/plugin.py

```diff
@@ -286,6 +291,10 @@ def _startup(self):
         if not self._ez_client.can_connect():
             raise RestConnectionError("Cannot connect to the Beer-garden server")
 
+        # If namespace couldn't be determined at init try one more time
+        if not self._config.namespace:
+            self._setup_namespace()
+
```

Comment:
In the code on line 302, should we be using namespace when defining the working directory? In a valid environment, a single BG server could be running identical plugins on different name spaces locally.

Reply:
Yes, that's a great catch.

Reply:
Should be resolved by 69513f6.